packaging/hip-targets-release.cmake

```diff
@@ -7,30 +7,40 @@ set(CMAKE_IMPORT_FILE_VERSION 1)
 
 # Set ROCM PATH from envinorment and default to /opt/rocm if not provided.
 if( DEFINED ENV{ROCM_PATH} )
-     set(ROCM_PATH $ENV{ROCM_PATH})
-elseif(DEFINED ENV{HIP_PATH})
-     set(ROCM_PATH "$ENV{HIP_PATH}/..")
-else ()
-     set(ROCM_PATH "/opt/rocm")
+     set(ROCM_PATH "$ENV{ROCM_PATH}")
 endif()
 
+get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" REALPATH)
+# Set ROCM PATH from envinorment and default to /opt/rocm if not provided.
+set (HIP_ROOT_DIR)
+find_path(
+          HIP_ROOT_DIR
```

Comment:
Find should be used to find the hip's root path. This always be relative to the cmake file. Find is only needed for hip's dependencies(such as hsa runtime).